src/main/c/Jep/pyembed.c

```diff
@@ -450,6 +449,10 @@ void pyembed_startup(JNIEnv *env, jobjectArray sharedModulesArgv)
                 PyEval_ReleaseThread(mainThreadState);
                 (*env)->PopLocalFrame(env, NULL);
                 THROW_JEP(env, "Received null argv.");
+                for(j = 0; j < i; j++) {
```

Comment:
Good catch.  Side note, I hate the letter `j` because it looks so close to `i` at times.  That caught me once and I swore never again so I use `k`.